enterprise/migrations/0114_enterpriseanalyticsuser_historicalenterpriseanalyticsuser.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 2.2.16 on 2020-10-19 11:15
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import model_utils.fields
+import simple_history.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('enterprise', '0113_auto_20200914_2054'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='HistoricalEnterpriseAnalyticsUser',
+            fields=[
+                ('id', models.IntegerField(auto_created=True, blank=True, db_index=True, verbose_name='ID')),
+                ('created', model_utils.fields.AutoCreatedField(default=django.utils.timezone.now, editable=False, verbose_name='created')),
+                ('modified', model_utils.fields.AutoLastModifiedField(default=django.utils.timezone.now, editable=False, verbose_name='modified')),
+                ('analytics_user_id', models.CharField(help_text='User id in the third party analytics system.', max_length=255)),
```

Comment:
We might want to change this from 'analytics_user_id' to 'tableau_username' to be more explicit about its usage. Tableau uses the term 'username' when discussing this kind of integration, so we should be consistent.

'analytics_user_id' is a good generic term, but we know that we are not very likely to be integrating with a third party other than Tableau, as opposed to the taxonomy-connector, in which we are expecting to integrate third parties other than EMSI.

Reply:
@moconnell1453 There are two fields in tableau ... username and user id ... the above represents the id ... which is essentially a uuid(36 characters). Who knows what the future may hold! ... lets keep the id generic.